drivers/python/rethinkdb/_restore.py

```diff
@@ -147,7 +147,7 @@ def parse_path(member):
         path, base = os.path.split(path)
         return (base, db, os.path.splitext(table_file)[0])
 
-    is_fileobj = type(options["in_file"]) is file
```

Comment:
For this case I think it would be a lot clearer to do this: `options["in_file"] is sys.stdin`. That does not allow for other open files, but in this case we don't allow for that anyways, and would need some more work to avoid copying the file anyways. Additionally I would rather see this line folded into line 155, as that would make it a more linear read.